### PR TITLE
[feat] - implement json merging algorithm POC

### PR DESCRIPTION
**Pull request by SHAKOTN**
## Done:
1. Walks through all dirs in `BIPs/` directory
2. Finds out what BIPs have `createdAt` within current week
3. Group them all by the chain and by msig address
4. In the end, collect all transactions from grouped BIPs and store to files in format `<chain_id>-<msig_addr>`

### CODE REVIEW

File: action-scripts/merge_pr_jsons.py [hunk 1]
```diff
@@ -23,43 +22,96 @@
...
-        safe = data["meta"]["createdFromSafeAddress"]
-        print(safe)
-        if safe not in txmap[chain_id].keys():
-            txmap[chain_id][safe] = []
-        txmap[chain_id][safe] = [*txmap[chain_id][safe], *data["transactions"]]
-        print(txmap)
+    current_week = datetime.utcnow().strftime("%U")
+    current_year = datetime.utcnow().year
+    # get root directory of the project:
+    root_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```
Comment (Tritium-VLK) [inline action-scripts/merge_pr_jsons.py:63]: This is hard for me/anyone who isn't a real pythonista to understand.  Can you provide a comment?

Comment (Tritium-VLK) [reply action-scripts/merge_pr_jsons.py:63]: is this just pwd?  Why so many nested calls?

File: action-scripts/merge_pr_jsons.py [hunk 2]
```diff
@@ -23,43 +22,96 @@
...
+    bips_dir = os.path.join(root_dir, "BIPs")
+    dir_with_target_bips = os.path.join(bips_dir, TARGET_DIR_WITH_BIPS)
```
Comment (Tritium-VLK) [inline action-scripts/merge_pr_jsons.py:65]: BIPs should be read from a path provided as an input for now (via env var or argv)

BIPs should be written to 00merged/YYYY-#WW  (or 00batched, there's one example there)

Comment (Tritium-VLK) [reply action-scripts/merge_pr_jsons.py:65]: the destination directory for the given run won't exist when the report runs.

Comment (SHAKOTN) [reply action-scripts/merge_pr_jsons.py:66]: That's clear. Gonna fix this

File: action-scripts/merge_pr_jsons.py (hunk 1 above)
Comment (SHAKOTN) [reply action-scripts/merge_pr_jsons.py:64]: Sure. When you call `os.path.abspath(__file__)` that would be full absolute path to the executed python file. Like:
```
os.path.abspath(__file__) == ''opt/multisig-ops/action-scripts/merge_pr_jsons.py''
```

Then you just want to move up 2 times, for that you are using `os.path.dirname`, and you end up in `opt/multisig-ops/`, which is a root directory of your project

Comment (SHAKOTN) [reply action-scripts/merge_pr_jsons.py:67]: Comment added

File: action-scripts/merge_pr_jsons.py (hunk 2 above)
Comment (SHAKOTN) [reply action-scripts/merge_pr_jsons.py]: Done

**Pull request merged**